R/rmarkdown.qmd

```diff
@@ -0,0 +1,80 @@
+# Rmarkdown {.unnumbered}
+
+The RStudio Docs have an excellent example [here](https://docs.rstudio.com/how-to-guides/users/basic/publish-rmd/) of how to create an Rmarkdown document using the built-in file creation tool and then publish it to Connect.
+
+An alternative method is shown below which is code based.
+
+## Prerequisites
+
+-   `rmarkdown` package must be installed
+
+## Create Content {.numbered}
+
+### 1. Create .Rmd file
+
+Create a new file in a text editor named, `rmarkdown_example.Rmd` that contains the following code.
+````markdown
```

Comment:
checkout `echo = "fenced"` https://mine-cetinkaya-rundel.github.io/quarto-tip-a-day/posts/02-fenced-echo/

Reply:
I did look this up.  It's an interesting option, but does not display the code in the way that I wanted.